### PR TITLE
education/connect mms api down to service-portal ui

### PR DESCRIPTION
**Pull request by darrikonn**
# Connect mms api to ui

(purposely didn't do any UI tweaks (e.g. the date in the card) since that will be done in another slice when we fix the logo + empty state) 

## Screenshots / Gifs

mms

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/api/src/app/environments/environment.ts [hunk 1]
```diff
@@ -11,11 +11,8 @@ export default {
     secret: process.env.DRIVING_LICENSE_SECRET,
   },
   education: {
-    xroadLicenseServiceId: 'r1/IS-DEV/EDU/10020/MMS-Protected/license-api-v1',
-    emailOptions: {
-      sendFromEmail: 'development@island.is',
-      useTestAccount: true,
-    },
+    xroadLicenseServiceId: 'IS-DEV/EDU/10020/MMS-Protected/license-api-v1',
+    fileDownloadBucket: 'island-is-dev-download-cache-api',
```
Comment (dabbeg) [inline apps/api/src/app/environments/environment.ts:15]: maybe name this something related to the project, education license something something

Comment (darrikonn) [reply apps/api/src/app/environments/environment.ts:15]: No this is a general file downloading bucket.

File: libs/api/domains/education/src/lib/education.service.ts [hunk 2]
```diff
@@ -1,71 +1,30 @@
...
-            encoding: 'binary',
-          },
-        ],
-      })
-    } catch (error) {
-      this.logger.error(`Failed to send email to ${email}`, error)
-      return null
-    }
-    return { email }
+  ): Promise<Response> {
```
Comment (dabbeg) [inline libs/api/domains/education/src/lib/education.service.ts:27]: FYI: no need to put return types on functions that return a typed object

Comment (darrikonn) [reply libs/api/domains/education/src/lib/education.service.ts:27]: Ahh ok, didn't know that :+1:

**Review by dabbeg — APPROVED** [review]
:+1:

**Pull request merged**